admin_guide/managing_projects.adoc

```diff
@@ -94,3 +101,40 @@ or:
 To request a new project, fill out the project request form located at
 https://internal.example.com/openshift-project-request.
 ----
+
+== Node Selectors
+
+Node selectors are used in conjunction with labels to control pod placement. You can assign a pod or set of pods to specific nodes, or limit a project to only run on a desired node.
```

Comment:
I would say a set of nodes. The set might only have one node, but we really don't want to encourage people to direct a pod to one specific node, because if that node goes down, their pod can't be rescheduled.